jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/perspectives/ProcessDefinitionsPerspective.java

```diff
@@ -44,4 +44,9 @@ public String getPerspectiveId() {
     public String getBasicFiltersScreenId() {
         return null;
     }
+
+    @Override
+    public String getSavedFiltersScreenId() {
+        return null;
```

Comment:
I'm, in general, against returning ```null```, but maybe this is a good case for an exception to that rule.
I can't think of any other approach except for introducing ```Optional``` as a return value, but that might just complicate the code and make it harder to read.

Reply:
+1